packages/wxt/src/cli/commands.ts

```diff
@@ -37,13 +44,9 @@ cli
         debug: flags.debug,
         filterEntrypoints: getArrayFromFlags(flags, 'filterEntrypoint'),
         dev:
-          flags.port == null
+          Object.keys(serverOptions).length === 0
             ? undefined
```

Comment:
There's a test which requires this to be `undefined` when neither `host` nor `port` are set